src/ansys/tools/installer/installed_table.py

```diff
@@ -300,23 +300,45 @@ def __init__(self, parent=None):
         hbox_install_pyansys = QtWidgets.QHBoxLayout()
         pyansys_pkg_manage_box_layout.addLayout(hbox_install_pyansys)
 
+        package_layout = QtWidgets.QVBoxLayout()
+        package_label = QtWidgets.QLabel("Package")
         self.model = QStandardItemModel()
         self.packages_combo = QComboBox()
         self.packages_combo.setModel(self.model)
+        package_layout.addWidget(package_label)
+        package_layout.addWidget(self.packages_combo)
 
+        version_layout = QtWidgets.QVBoxLayout()
+        version_label = QtWidgets.QLabel("Version")
         self.versions_combo = QComboBox()
+        version_layout.addWidget(version_label)
+        version_layout.addWidget(self.versions_combo)
+
+        target_layout = QtWidgets.QVBoxLayout()
+        target_label = QtWidgets.QLabel("Target")
```

Comment:
```suggestion
        target_label = QtWidgets.QLabel("Target (optional)")
```
@anssakthi @tusharbana-ansys -- are you okay with this approach?

Reply:
I am okay with this change